doc/source/contribution/documentation/introduction.rst

```diff
@@ -38,7 +42,7 @@ are installation steps based on `pyenv
 
   % pyenv install 2.7.11
   % pyenv global 2.7.11
-  % pip install sphinx
+  % pip install -r doc/requirements.txt
```

Comment:
We can remove this pyenv related section. All major platforms must provide Python 3 now.